torchvision/transforms/functional_pil.py

```diff
@@ -118,7 +118,7 @@ def adjust_saturation(img, saturation_factor):
 
 
 @torch.jit.unused
-def adjust_hue(img, hue_factor):
+def adjust_hue(img, hue_factor: float):
```

Comment:
This is not necessary. Otherwise, you can set it as `def adjust_hue(img: Any, hue_factor: float):`